src/angularComponent/ngcOmniboxSuggestionItemDirective.js

```diff
@@ -0,0 +1,18 @@
+import NgcOmniboxSuggestionItemController from './ngcOmniboxSuggestionItemController.js';
+
+export default function ngcOmniboxSuggestionsDirective() {
+  return {
+    restrict: 'AE',
+    require: '^^ngcOmnibox',
```

Comment:
We should also make sure this also requires `ngc-omnibox-suggestions` to be a parent, just to make sure people don't forget to make that element.